src/main/java/hudson/plugins/parameterizedtrigger/DefaultParameterValuesActionsTransform.java

```diff
@@ -1,11 +1,6 @@
 package hudson.plugins.parameterizedtrigger;
 
-import hudson.model.AbstractProject;
-import hudson.model.Action;
-import hudson.model.ParametersAction;
-import hudson.model.ParameterDefinition;
-import hudson.model.ParameterValue;
-import hudson.model.ParametersDefinitionProperty;
+import hudson.model.*;
```

Comment:
There is no general styleguide for Jenkins plugins. My personal preference is to avoid star imports (except for `import static`), and keep all imports simply sorted in one block—after all, imports are for `javac` and your IDE to read, not for humans.